src/services/osmApi.ts

```diff
@@ -94,10 +94,38 @@ const getCountryCode = async (feature: Feature): Promise<string | null> => {
   return null;
 };
 
+const getElementsAndCenter = async (
+  apiId: OsmId,
+): Promise<{ element: any; center: false | LonLat }> => {
+  const elementPromise = getOsmPromise(apiId);
```

Comment:
Can you please remove this variable and inline the call into the usages?

Extracting makes sense for bigger poritions of code - to make it more readable. But this works contrary - you have to read an identifier to get what simple code it makes. Even though, you could have read the call itself.